### PR TITLE
build: add more files to .gitignore

### PR DESCRIPTION
**Pull request by pkalever**
added
1. generated files
2. cscope & ctags generated files and
3. patch files

Signed-off-by: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>

### CODE REVIEW

**Comment by pkalever** [comment]
please-review: @lpabon @obnoxxx @raghavendra-talur

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,13 @@ heketi.db
...
+
+# Generated files
+heketi
+client/cli/go/heketi-cli
+
+# Other
+cscope.*
+tags
+*.diff
+*.patch
```
Comment (obnoxxx) [inline .gitignore]: `*.patch` is already listed above.

Comment (obnoxxx) [inline .gitignore]: Apart from that -- looks good to me.
I'd like to add vim files: `*~` and `*.swp` .

**Comment by pkalever** [comment]
@obnoxxx Thanks!
Please check the updated one!

**Comment by obnoxxx** [comment]
Looks good to me, but the two patches should be squashed, imho. Thanks!

**Comment by lpabon** [comment]
The unit tests failed for another reason.  I created #483 to track it

**Pull request merged**